test/src/main/java/org/apache/accumulo/test/functional/CompactionIT.java

```diff
@@ -1067,21 +1067,26 @@ public void testOfflineAndCompactions() throws Exception {
 
       // start a bunch of compactions in the background
       var executor = Executors.newCachedThreadPool();
-      List<Future<?>> futures = new ArrayList<>();
+      final int numTasks = 20;
+      List<Future<?>> futures = new ArrayList<>(numTasks);
+      CountDownLatch startLatch = new CountDownLatch(numTasks);
       // start user compactions on a subset of the tables tablets, system compactions should attempt
       // to run on all tablets. With concurrency should get a mix.
-      for (int i = 1; i < 20; i++) {
+      for (int i = 0; i < numTasks; i++) {
         var startRow = new Text(String.format("r:%04d", i - 1));
         var endRow = new Text(String.format("r:%04d", i));
```

Comment:
Seems like this change from i starting at 1 to 0 might be an issue with the (i - 1). May want to change this back or address this (i - 1) in some way.
```suggestion
      for (int i = 1; i < numTasks + 1; i++) {
        var startRow = new Text(String.format("r:%04d", i - 1));
        var endRow = new Text(String.format("r:%04d", i));
```